### ISSUE TITLE
Website should contain contacts for rectifiying erroneous bans

### ISSUE THREAD

**Issue by safield** [the-programmers-hangout/website#227]
I recently tried joining the programmers hangout discord channel. It seemed to join successfully. I then went afk, and a timed human verification procedure timed out, and automatically banned my account.

Now I can not find any resources or contacts to help in rectifying this issue. I imagine this is a common issue, there should be someway present on the website to help rectify this issue.

**Comment by veksen** [comment]
Totally, and thank you for the feedback! Brainstorming something to ease this.

**Comment by resynth1943** [comment]
Yeah, totally! I was banned for insulting Fox in another server, and I still haven't heard anything back. Do you have a feedback system?

**Comment by veksen** [comment]
This is in the works, and will be included in the website!

**Comment by resynth1943** [comment]
@veksen This will be useful for new members that have been banned unfairly.
